lm_eval/tasks/MedArabiQ/utils.py

```diff
@@ -0,0 +1,32 @@
+try:
+    import evaluate
+except ImportError:
+    print("The 'evaluate' library is not installed. Please install it using `pip install evaluate`.")
+    exit(1)
+
+
+def doc_to_choice(doc):
+    return [option[0] for option in doc["options"]]
+
+def doc_to_target(doc):
+    options = doc_to_choice(doc)
+    return options.index(doc["correct_answer"])
+
+
+def bert_score(items): return items
+
+def agg_bert_score(items):
+    print("Computing BERT score...")
+    # chose this model due to its small size compared to its good performance
```

Comment:
* Remove these comments (non relevant to your chosen model)
* Add antother comment stating why you chose this embedding model (because it's the same model used in the paper) and add the link to the paper